packages/__docs__/src/Properties/index.tsx

```diff
@@ -175,9 +175,7 @@ class Properties extends Component<PropertiesProps> {
     if (prop.required) {
       return <span css={styles?.required}>Required</span>
     } else if (prop.defaultValue) {
-      let defaultValue: string | React.ReactNode = this.unquote(
-        prop.defaultValue.value
-      )
+      let defaultValue: string | React.ReactNode = prop.defaultValue.value
```

Comment:
IMO removing the quotes is not needed, its now easier to see whether a value is a string or number. Also the unquote method is sometimes buggy and just removes one quote sign, e.g. in https://instructure.design/#AppNav -> `renderTruncateLabel`